Services/CO.CDP.OrganisationInformation.Persistence/Forms/FormAnswer.cs

```diff
@@ -20,6 +20,7 @@ public class SharedConsent : IEntityDate
     public DateTimeOffset? SubmittedAt { get; set; }
     public required string FormVersionId { get; init; }
     public string? BookingReference { get; set; }
+    public string? ShareCode { get; set; }
```

Comment:
Is ShareCode different than BookingReference?

Reply:
Good Point @dharmverma  
we have a share code in BookingReference
![image](https://github.com/user-attachments/assets/ec346394-12c5-4ce8-a5ee-4b941052a936)

Reply:
It is not. We already have BookingReferecen, so either use BookingReference, or replace BookingReference with ShareCode.

If you decided to replace BookingReference with ShareCode, I'd prefer to have it done in a separate PR as it impacts few other things in progress.